contrib/IECoreArnold/src/IECoreArnold/UniverseBlock.cpp

```diff
@@ -80,6 +80,10 @@ void loadMetadata( const std::string &pluginPaths )
 void begin()
 {
 	AiBegin();
+	// Default to logging errors / warnings only - we may not even be using this universe block to perform a render,
```

Comment:
So Arnold 5 defaults to _all_ logging being on?

Reply:
Not sure if it is all logging, maybe it was reporting the paths it was loading shaders from into "info"?  It was a lot of annoying output every time you instantiated a universe block anyway